models/base/scratch/snowplow_normalize_base_events_this_run.sql

```diff
@@ -14,24 +14,37 @@ You may obtain a copy of the Snowplow Personal and Academic License Version 1.0
 
 {%- set lower_limit, upper_limit, session_start_limit = snowplow_utils.return_base_new_event_limits(ref('snowplow_normalize_base_new_event_limits')) %}
 
-select
-    a.*
+with final_cte AS (
 
-from {{ var('snowplow__events') }} as a
+  select
+      a.*
 
-where
-  {# dvce_sent_tstamp is an optional field and not all trackers/webhooks populate it, this means this filter needs to be optional #}
-  {% if var("snowplow__days_late_allowed") == -1 %}
-    1 = 1
-  {% else %}
-    a.dvce_sent_tstamp <= {{ snowplow_utils.timestamp_add('day', var("snowplow__days_late_allowed", 3), 'a.dvce_created_tstamp') }}
+  {% if target.type not in ['databricks','snowflake','bigquery'] %}
```

Comment:
NIT again, but can we add a developer comment above this that spark doesn't support qualify hence the additional logic for future reference? It will get lost outside the context of a spark update, that's all.

Reply:
Agree!! Ultra fair point

Reply:
Can we change the -- to {# #} so that it does not get compiled to the users?

Reply:
Roger!!